packages/components/src/DataList/DataList.utils.tsx

```diff
@@ -89,13 +89,7 @@ export function generateHeaderElements<T extends DataListObject>(
   const headerElements = Object.keys(headers).reduce(
     (acc, key) => ({
       ...acc,
-      [key]: (
-        <div className={styles.headerLabel}>
-          <Text variation="subdued" maxLines="single" size="small">
-            {headers[key]}
-          </Text>
-        </div>
-      ),
+      [key]: <DataListHeaderTile headers={headers} headerKey={key} />,
```

Comment:
can grab that `headers` prop from the context, no?

Reply:
oh yeah, good catch!